src/test/crypto_tests.cpp

```diff
@@ -555,6 +555,7 @@ BOOST_AUTO_TEST_CASE(poly1305_testvector)
 
 BOOST_AUTO_TEST_CASE(countbits_tests)
 {
+    FastRandomContext ctx;
```

Comment:
Redundant if L571 is changed to use `InsecureRandBits` instead.

Reply:
Fixed.  Interestingly, this exists upstream